checkbox-ng/checkbox_ng/version.py

```diff
@@ -31,63 +31,30 @@
 
 
 def get_version_info():
-    info = OrderedDict()
-    if os.environ.get("SNAP"):
-        return get_parts_info()
-    else:
-        return get_stack_version()
-
-
-def get_stack_version():
     """
     Generate version information of Checkbox stack components as understood
     by Checkbox.
     """
     info = OrderedDict()
-    info['checkbox-ng'] = checkbox_ng.__version__
+    info["checkbox-ng"] = checkbox_ng.__version__
     try:
         import checkbox_support
```

Comment:
Let's just report the version of checkbox-ng here.

Reply:
Agree